packages/block-library/src/post-featured-image/dimension-controls.js

```diff
@@ -143,6 +146,30 @@ const DimensionControls = ( {
 					</ToggleGroupControl>
 				</ToolsPanelItem>
 			) }
+			{ !! imageSizeOptions.length && (
+				<ToolsPanelItem
+					hasValue={ () => !! sizeSlug && sizeSlug !== DEFAULT_SIZE }
+					label={ __( 'Size' ) }
```

Comment:
Would it make sense to use `Image size` rather than `Size`? That matches the labeling in the Image block for example:
<img width="275" alt="Screen Shot 2022-01-19 at 10 51 51 AM" src="https://user-images.githubusercontent.com/63313398/150197367-e8604c34-083a-4db0-aba8-b38ecdf4e271.png">

The added helptext is great in the panel, but it feels a bit confusing to see `Size` listed alongside `Width` and `Height` in the ToolsPanel _menu_ where you lack that context.

<img width="279" alt="Screen Shot 2022-01-19 at 10 52 22 AM" src="https://user-images.githubusercontent.com/63313398/150197899-9a3c066d-bc49-4311-aba1-2a9dc0d9fb06.png">

Reply:
I don't have any preference here, but since we're trying to unify image dimensions controls across the blocks would be nice to agree on a single `ToolsPanelItem` label.


@jasmussen, @jameskoster, what do you think?

Reply:
I have updated the label. I'm going to merge this, and we can change labels back after settings unification.